Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -398,44 +404,46 @@ final class StatusTableViewController: ChartsTableViewController {
 
             // Glucose
             if let glucoseValues = glucoseValues {
-                self.charts.setGlucoseValues(glucoseValues)
+                self.statusCharts.setGlucoseValues(glucoseValues)
             }
             if let predictedGlucoseValues = predictedGlucoseValues {
-                self.charts.setPredictedGlucoseValues(predictedGlucoseValues)
+                self.statusCharts.setPredictedGlucoseValues(predictedGlucoseValues)
             }
-            if let lastPoint = self.charts.predictedGlucosePoints.last?.y {
+            if let lastPoint = self.statusCharts.glucose.predictedGlucosePoints.last?.y {
                 self.eventualGlucoseDescription = String(describing: lastPoint)
             } else {
                 self.eventualGlucoseDescription = nil
             }
             if currentContext.contains(.targets) {
-                self.charts.targetGlucoseSchedule = self.deviceManager.loopManager.settings.glucoseTargetRangeSchedule
+                self.statusCharts.targetGlucoseSchedule = self.deviceManager.loopManager.settings.glucoseTargetRangeSchedule
             }
 
+            let charts = self.statusCharts
+
             // Active Insulin
             if let iobValues = iobValues {
-                self.charts.setIOBValues(iobValues)
+                charts.setIOBValues(iobValues)
             }
-            if let index = self.charts.iobPoints.closestIndexPriorToDate(Date()) {
-                self.currentIOBDescription = String(describing: self.charts.iobPoints[index].y)
+            if let index = charts.iob.iobPoints.closestIndexPriorToDate(Date()) {
```

Comment:
Naming nit while we're in refactoring mode:
`closestIndexPriorToDate(_:)` -> `closestIndex(priorTo:)`